site2/docs/client-libraries-python.md

```diff
@@ -9,29 +9,21 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 ````
 
+You can use a Pulsar Python client to create producers, consumers, and readers.
 
-Pulsar Python client library is a wrapper over the existing [C++ client library](client-libraries-cpp.md) and exposes all of the [same features](/api/cpp). You can find the code in the [Python directory](https://github.com/apache/pulsar/tree/master/pulsar-client-cpp/python) of the C++ client code.
+All the methods in producer, consumer, and reader of a Python client are thread-safe. You can read the [API docs](/api/python) for the Python client.
 
-All the methods in producer, consumer, and reader of a Python client are thread-safe.
+## Installation
 
-pdoc-generated API docs for the Python client are available [here](/api/python).
+Use [pip](https://pip.pypa.io/) to install the latest version:
```

Comment:
Is this the only way to install a Python client?

Reply:
Yes. And installing library via `pip` is the idiom in Python community. Although documenting how to build from source can help for hackers, this is currently impossible because of https://github.com/apache/pulsar/pull/17796#discussion_r977310599.